src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  background-color: #{$White};
+  border: 1px solid #f3f3f3;
+  border-radius: #{$Border-radius};
+  width: 200px;
+  padding: 2rem 1rem 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+
+  .card__img {
+    height: 134px;
+    object-fit: contain;
+    margin-bottom: 2.5rem;
+  }
+  
+  .card__name {
+    font-size: 0.75rem;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 0.25rem;
+    color: #{$Main-accent};
+  }
+  
+  .card__code {
+    margin-bottom: 1rem;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: #{$Secondary};
+  }
+  
+  .card__ratings {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 1.5rem;
+  
+    .card__reviews {
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 14px;
+      color: #{$Main-accent};
+    }
+  }
+  
+  .card__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 1rem;
+  
+    span:first-child {
+      font-size: 0.75rem;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px;
+      color: #{$Secondary};
+    }
+  
+    span:last-child {
+      font-size: 1rem;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 18px;
+      color: #{$Main-accent};
+    }
+  
+  }
+  
+  .card__link {
+    width: 100%;
+    padding: 0.75rem 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 1rem;
+    text-transform: uppercase;
+    color: #{$White};
+    border: 1px solid transparent;
+    border-radius: #{$Border-radius};
+    background: #{$Blue-accent};
+
+    &:hover {
+      background-color: $White;
+      color: $Blue-accent;
+      border: 1px solid $Blue-accent;
```

Comment:
I know that I'm being picky right now but you could change only the border color.